README.md

```diff
@@ -1,13 +1,43 @@
-# CorePPL
-To be written...
+# Miking DPPL
+Miking DPPL is a framework for developing probabilistic programming languages (PPLs) using [Miking](https://github.com/miking-lang/miking).
+Currently, the framework includes the PPLs [CorePPL](#coreppl) and [RootPPL](#rootppl).
 
-# RootPPL
-RootPPL can be seen as an intermediate language for representing probabilistic models and comes with a framework that performs inference on the GPU in these models. These models are currently hard-coded, see examples in the folder rootppl/models. The idea is that high-level Miking probabilistic programming languages should be compiled to this intermediate language. 
+## Building and Installing
+The main binary for Miking DPPL is called `midppl`. Currently, this binary is built by running `make` in the project root, and installed to `$HOME/.local/bin` with `make install` (uninstall is also possible through `make uninstall`). You must have [Miking](https://github.com/miking-lang/miking) installed (the `mi` command must be globally available) for building `midppl`.
+The command `make test` executes a set of tests for various components in the Miking DPPL framework.
 
-## Getting Started
+## CorePPL
+CorePPL extends MExpr (see [Miking](https://github.com/miking-lang/miking)) with probabilistic constructs for defining random variables and for likelihood updating. For example, the program
+```
+mexpr
+let x = assume (Beta 10.0 5.0) in
+observe true (Bernoulli x);
+x
+```
+encodes a simple distribution for the bias of a coin, by setting a Beta prior for the bias and observing a single flip of the coin (resulting in `true`, i.e. heads).
```

Comment:
From earlier discussions, we should probably not use the term `bias` in this context. Suggestion: "... by setting a Beta prior for the probability observing head (resulting in `true`) for a single flip of the coin."

Reply:
Done, resolving